suite-native/app/e2e/tests/appSettings.test.ts

```diff
@@ -43,6 +45,17 @@ describe('App Settings - without device interactions [@noDevice]', () => {
         await detoxExpect(element(by.text('0 sat'))).toBeVisible();
     });
 
+    it('Localization - Language', async () => {
+        await onTabBar.assertHomeTabBarItemTitle(EN_TRANSLATIONS['navigation.tabs.home']);
```

Comment:
(still learning) would this work?
```suggestion
        await detoxExpect(element(by.id('@tabBar/HomeStack/title'))).toHaveText(EN_TRANSLATIONS['navigation.tabs.home']);
```

Reply:
it should I fixed it inside of the `assertHomeTabBarItemTitle` method. Fixed in 64b2f6732e

Reply:
generally detox recommends to use expect over waitFor where possible :+1: